src/Microsoft.Azure.CosmosRepository/Services/DefaultCosmosContainerService.cs

```diff
@@ -60,10 +63,13 @@ private async Task<Container> GetContainerAsync(Type itemType, bool forceContain
                 };
 
                 Container container =
-                    await database.CreateContainerIfNotExistsAsync(
-                        containerProperties, itemConfiguration.ThroughputProperties).ConfigureAwait(false);
+                    _options.IsAutoResourceCreationIfNotExistsEnabled
+                        ? await database.CreateContainerIfNotExistsAsync(
+                            containerProperties, itemConfiguration.ThroughputProperties).ConfigureAwait(false)
+                        : await Task.FromResult(database.GetContainer(containerProperties.Id)).ConfigureAwait(false);
```

Comment:
```suggestion
                    _options.IsAutoResourceCreationIfNotExistsEnabled
                        ? await database.CreateContainerIfNotExistsAsync(
                            containerProperties, itemConfiguration.ThroughputProperties).ConfigureAwait(false)
                        : await Task.FromResult(database.GetContainer(containerProperties.Id));
```

Do we need to call `ConfigureAwait` on the from result calls?